tests/quasi-newton/serial/TestQN11.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <precice-configuration experimental="true">
-  <data:scalar name="Data1" waveform-degree="2" />
-  <data:scalar name="Data2" waveform-degree="2" />
+  <data:scalar name="Data1" waveform-degree="1" />
+  <data:scalar name="Data2" waveform-degree="1" />
```

Comment:
Is this change necessary? I do not really see how this related to the regression.